src/dataMap/metaManager/metaSchema.js

```diff
@@ -740,6 +767,7 @@ export default () => {
      * @memberof Options#
      * @type {boolean|string|object}
      * @default true
+     * ! TODO @category (there is no fillHandle plugin in the `src/plugins`)
```

Comment:
The Autofill plugin uses it to check if it is enabled.

![image](https://user-images.githubusercontent.com/2893099/115539320-2046ad00-a29d-11eb-8f12-381c46bd2605.png)

Why do I think it is not a MultipleSelect? Because of usage context.